.github/workflows/webapp-ci-pr.yml

```diff
@@ -1,6 +1,8 @@
 name: Web App CI PR
 on:
   pull_request:
+    paths:
+      - 'webapp/**/*'
```

Comment:
`**` already matches everything. Do we need to add a `/*` after that?

Reply:
Good catch. I think that's left over from me trying to do something like `**/*.js`